### ISSUE TITLE
Is it okay to use the assembled contig and metagenome reads as the input of the METABOLIC?

### ISSUE THREAD

**Issue by B-1991-ing** [AnantharamanLab/METABOLIC#95]
Hi Zhichao,

I got 55 MAGs from the 6 metagenome samples, but some typical genes for some MAGs were lost maybe during the binning or refinement process. So, is it okay that I use the assembled contigs and metagenome fq reads as the input of the METABOLIC to see the metabolic cycling process at the community level? Does it make sense to do it this way?

Best,

Bing

**Comment by B-1991-ing** [comment]
In my condition, contigs from the co-assembly are all in one fasta file for multiple metagenome samples, but contigs from the individual-assembly are all in each separate fasta file for each metagenome sample. I think it is possible to have the gtdbtk classification for each fasta file which contains many contigs?

Best, 

Bing

**Comment by ChaoLab** [comment]
> In my condition, contigs from the co-assembly are all in one fasta file for multiple metagenome samples, but contigs from the individual-assembly are all in each separate fasta file for each metagenome sample. I think it is possible to have the gtdbtk classification for each fasta file which contains many contigs?
> 
> Best,
> 
> Bing

The assembly from each metagenome sample contains sequences from many microbial genomes. GTDB-Tk will treat a single fast file as one genome. It will not give a meaningful taxonomy

**Comment by haihao999** [comment]
Hi,zhichao
After refining the genome,Is there a value that needs to be set for completeness and contamination ? e.g. completss>50% con<10%

**Comment by ChaoLab** [comment]
Yes, completeness >= 50%, contamination < 10% will be a minimal standard to refine a genome

**Comment by haihao999** [comment]
Thank you ,zhichao
I would like to confirm another question，Is it possible to analyze the genome based on the public database and the genome I got from binning after redundancy？

**Comment by ChaoLab** [comment]
METABOLIC will annotate genomes using multiple databases in a comprehensive manner. It is mainly based on using HMM search

**Comment by haihao999** [comment]
hi，zhichao
I mean, my genome is not obtained from my personal sample, rather it is a publicly available genome used to analyze my sample.
Is this feasible? Thank you

**Comment by ChaoLab** [comment]
I am not so aware of what you are doing. If you are using some others' datasets, better to be sure on the rights to use the data
